tests/helpers.py

```diff
@@ -30,3 +30,7 @@ def time_spent(function, *args, **kwargs):
     end_time = time.time()
 
     return end_time - start_time
+
+
+def _set_to_ms(timeout):
```

Comment:
Since it's not a class method it can be named without `_`.
Also the function name could be more descriptive like `convert_sec_to_ms`.

Reply:
Done

Reply:
👍

Reply:
Good job @Vikilib!